refactor(ui): add latest labels for Italian language localization

polilluminato opened the pull request:
Italian translation

russellwheatley commented:
Hey @polilluminato, do you mind fixing the formatting and analyse CI issues so we can get this merged in, please?

Salakar commented on packages/flutterfire_ui/lib/src/i10n/lang/it.dart:
```diff
@@ -251,14 +251,14 @@ class ItLocalizations extends FlutterFireUILocalizationLabels {
+        'Una email di verifica è stata appena inviata al tuo indirizzo email. '
+            'Controlla la tua email e clicca sul link per la verifica '
+            'del tuo indirizzo email.',
+    this.couldntVerifyEmailText = "Non possiamo verificare il tuo indirizzo email. ",
```
```suggestion
    this.couldntVerifyEmailText = 'Non possiamo verificare il tuo indirizzo email.',
```

polilluminato commented:
> Hey @polilluminato, do you mind fixing the formatting and analyse CI issues so we can get this merged in, please?

I don't understand with I have to do. I saw that @Salakar already fix the format.

russellwheatley commented:
Hey @polilluminato, click on details next to the failiing CI tests here:
<img width="859" alt="Screenshot 2022-05-04 at 12 55 42" src="https://user-images.githubusercontent.com/16018629/166676440-6714354a-b722-4b5a-9c46-3f763e6fe832.png">

You can format a file on the command line by going to the directory and running `flutter format .`. You will also need to check the analyze issue and update based on feedback. Thank you.

polilluminato commented:
@russellwheatley I did it and pushed the code.

russellwheatley commented:
@polilluminato looks like you'll need to format more files  - `packages/flutterfire_ui/lib/src/i10n/lang` go to this directory and run this command `flutter format .`

Fix these analyse issues:

```bash
[flutterfire_ui]:    info - lib/src/i10n/lang/id.dart:260:35 - Only use double quotes for strings containing single quotes. - prefer_single_quotes
[flutterfire_ui]:    info - lib/src/i10n/lang/ja.dart:254:35 - Only use double quotes for strings containing single quotes. - prefer_single_quotes
```

Thanks!

polilluminato commented:
@russellwheatley I think I've done all the steps:

- [x] run `flutter format .` in `packages/flutterfire_ui/lib/src/i10n/lang`
- [x] fixed issue on lib/src/i10n/lang/id.dart
- [x] fixed issue on lib/src/i10n/lang/ja.dart

russellwheatley commented:
If you merge with master, the last analyze issue should disappear, @polilluminato 🙏

russellwheatley commented:
Something has gone wrong, @polilluminato 😓. PR now changes 466 files...

polilluminato commented:
> Something has gone wrong, @polilluminato sweat. PR now changes 466 files...

I think I've made a mess. Maybe it's a good idea to dismiss this PR and start from scratch?

russellwheatley commented:
Hey @polilluminato, if you wish. We'd still gratefully accept another PR submission 😄
